dashboards-observability/public/components/custom_panels/panel_modules/visualization_flyout/__tests__/__snapshots__/visualization_flyout.test.tsx.snap

```diff
@@ -92,7 +92,7 @@ exports[`Visualization Flyout Component renders add visualization Flyout 1`] = `
         ownFocus={false}
         size="m"
       >
-        <OuiWindowEvent
+        <EuiWindowEvent
```

Comment:
Revert these files as it will allow to pass the check on main.

Reply:
Reverted